inst/admiral-lookup-book.csv

```diff
@@ -95,245 +173,237 @@ ADLB,BDS,ASEQ,Derive sequential number for observations within each by group,adm
-ADPC,OCCDS,TRTSDTM,Derive treatment start datetime,admiral,derive_vars_merged,https://pharmaverse.github.io/admiral/reference/derive_vars_merged.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adpc.R
-ADPC,OCCDS,TRT01A,Derive actual treatment for period 1 ,admiral,derive_vars_merged,https://pharmaverse.github.io/admiral/reference/derive_vars_merged.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adpc.R
-ADPC,OCCDS,TRT01P,Derive planned treatment for period 1,admiral,derive_vars_merged,https://pharmaverse.github.io/admiral/reference/derive_vars_merged.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adpc.R
-ADPC,OCCDS,ADTM,Derive datetime from a date character vector,admiral,derive_vars_dtm,https://pharmaverse.github.io/admiral/reference/derive_vars_dtm.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adpc.R
-ADPC,OCCDS,ADTF,Derive date imputation flag,admiral,derive_vars_dtm,https://pharmaverse.github.io/admiral/reference/derive_vars_dtm.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adpc.R
-ADPC,OCCDS,ATMF,Derive time imputation flag,admiral,derive_vars_dtm,https://pharmaverse.github.io/admiral/reference/derive_vars_dtm.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adpc.R
+ADMH,OCCDS,NA,Derive query variables,admiral,derive_vars_query,https://pharmaverse.github.io/admiral/reference/derive_vars_query.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_admh.R
+ADMH,OCCDS,MHTERMN,Derive company specific medical history term variable,admiral,restrict_derivation,https://pharmaverse.github.io/admiral/reference/restrict_derivation.html,Template Example,https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_admh.R
+ADOE,ADSL,ABLFL,Derive analysis baseline flag,admiral,restrict_derivation,https://pharmaverse.github.io/admiral/reference/restrict_derivation.html,Template Example,https://github.com/pharmaverse/admiralophtha/blob/main/inst/templates/ad_adoe.R
+ADOE,ADSL,ADT,Derive analysis date,admiral,derive_vars_dt,https://pharmaverse.github.io/admiral/reference/derive_vars_dt.html,Template Example,https://github.com/pharmaverse/admiralophtha/blob/main/inst/templates/ad_adoe.R
+ADOE,ADSL,ADY,Derive analysis relative day,admiral,derive_vars_dy,https://pharmaverse.github.io/admiral/reference/derive_vars_dy.html,Template Example,https://github.com/pharmaverse/admiralophtha/blob/main/inst/templates/ad_adoe.R
+ADOE,ADSL,ANL01FL,Derive flag last result within a visit and timepoint for baseline and post-baseline records,admiral,restrict_derivation,https://pharmaverse.github.io/admiral/reference/restrict_derivation.html,Template Example,https://github.com/pharmaverse/admiralophtha/blob/main/inst/templates/ad_adoe.R
```

Comment:
```suggestion
ADOE,ADSL,ANL01FL,Derive flag for last result within a visit and timepoint for baseline and post-baseline records,admiral,restrict_derivation,https://pharmaverse.github.io/admiral/reference/restrict_derivation.html,Template Example,https://github.com/pharmaverse/admiralophtha/blob/main/inst/templates/ad_adoe.R
```

Reply:
Unsure on this one - others might need to be looked at as well

Reply:
@bms63 @ddsjoberg @athenamelia Could you please advise me on this one? Is the description correct or shall it be adjusted?